colplus-admin/src/main/java/org/col/admin/task/importer/acef/AcefDateParser.java

```diff
@@ -1,86 +1,13 @@
 package org.col.admin.task.importer.acef;
 
-import org.apache.commons.lang3.StringUtils;
-import org.col.parser.Parser;
-import org.col.parser.UnparsableException;
+import org.col.parser.FuzzyDateParser;
 
-import java.time.LocalDate;
-import java.time.Year;
-import java.time.YearMonth;
-import java.time.format.DateTimeFormatter;
-import java.time.format.DateTimeParseException;
-import java.util.Arrays;
-import java.util.List;
-import java.util.Optional;
-
-/*
- * Currently allows only exactly for those patterns actually encountered in the ACEF sources. We
- * could allow for more date patterns (e.g. month not case sensitive), or even allow the patterns to
- * be settable.
- */
-public class AcefDateParser implements Parser<LocalDate> {
+public class AcefDateParser extends FuzzyDateParser {
```

Comment:
Is it worth a class on its own?
I would have rather thought we have a single DateParser class with some specific factory methods or singletons that load the appropriate configs. Sth like:

class  DateParser {
  public static final ACEF = new DateParser("acef.fuzzy-date.properties")